### PR TITLE
Demo update for v4.3.0

### PR DESCRIPTION
**Pull request by fnevgeny**
I suggest not to keep the compiled demo under the version control.

### CODE REVIEW

**Review by AndreasBilke — CHANGES_REQUESTED** [review]
I agree on everything except the removal of demo/pdfpc-demo.pdf

You publish the pdfpc website by pushing your changes to the master branch. The GitHub magic now fetches your stuff and put it on there servers (without any further "make" step. it just copies the files).

So without the pdf under git, this file would be missing on the final web page.

File: demo/pdfpc-demo.tex [hunk 1]
```diff
@@ -27,7 +27,8 @@
 \mode<presentation>
 
 \title{\pdfpc demo}
-\author[D. Vilar]{David Vilar \\ \texttt{davvil@gmail.com}}
+\author[D. Vilar]{David Vilar \\ \texttt{davvil@gmail.com} \\
+                  Updated by E. Stambulchik}
```
Comment (AndreasBilke) [inline demo/pdfpc-demo.tex]: Either put your mail address there as well or remove david's?

Comment (fnevgeny) [reply demo/pdfpc-demo.tex]: OK, changed to github hyperlinks. I think nobody wants to get more spam from email web harvesters...

**Comment by fnevgeny** [comment]
> So without the pdf under git, this file would be missing on the final web page.

But the web page doesn't have to refer to this specific url. E.g., the compiled file can be made part of pdfpc release (as an asset). There is an [API](https://developer.github.com/v3/repos/releases/#upload-a-release-asset) to make this automatic, if desired.

In fact, this demo may become a kind of the official user guide; then it would make more sense to move it to the pdfpc repository, just like man pages. What do you think? Going one step further, the entire github.io site can be moved to the `doc` folder (which is a popular choice in many github projects). It is easier to keep everything in sync this way, IMO.

**Comment by AndreasBilke** [comment]
> > So without the pdf under git, this file would be missing on the final web page.
> 
> But the web page doesn't have to refer to this specific url. E.g., the compiled file can be made part of pdfpc release (as an asset). There is an [API](https://developer.github.com/v3/repos/releases/#upload-a-release-asset) to make this automatic, if desired.

Okay, that might be an option.

> In fact, this demo may become a kind of the official user guide; then it would make more sense to move it to the pdfpc repository, just like man pages. What do you think? Going one step further, the entire github.io site can be moved to the `doc` folder (which is a popular choice in many github projects). It is easier to keep everything in sync this way, IMO.

I'm also fine with that. I didn't liked the manual web page changes anyways :)

But for now: If I merge this PR, nothing would change on the website, since the new `pdfpc-demo.pdf` file is missing.

**Review by AndreasBilke — APPROVED** [review]

**Comment by fnevgeny** [comment]
> But for now: If I merge this PR, nothing would change on the website, since the new `pdfpc-demo.pdf` file is missing.

OK, let's add it this time manually to v4.3.0 as an asset and update the html links (also in README), and then we'll have time to discuss further?

**Comment by AndreasBilke** [comment]
> > But for now: If I merge this PR, nothing would change on the website, since the new `pdfpc-demo.pdf` file is missing.
> 
> OK, let's add it this time manually to v4.3.0 as an asset and update the html links (also in README), and then we'll have time to discuss further?

Done.

**Pull request merged**